.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: bash
+
+# Use container-based infrastructure for quicker build start-up
+sudo: false
```

Comment:
sudo: false means " Use container-based infrastructure for quicker build start-up"?